process-manager.js

```diff
@@ -88,38 +93,46 @@ class ProcessManager {
 	release(process) {
 		process.release();
 	}
+
 	send() {
 		if (!this.processes.length) {
 			return Promise.resolve(this.receive.apply(this, arguments));
 		}
 
+		// Prevents the arguments object from leaking.
+		let args = [];
+		for (let i = 0; i < arguments.length; i++) {
+			args.push(arguments[i]);
+		}
```

Comment:
We can hardcode arguments.

Just have the function signature be `send(arg1, arg2, arg3)`. I don't think we have a need for more than 3 arguments to a `send` function?